package.json

```diff
@@ -29,7 +29,7 @@
     "axios": "^1.6.0",
     "browser-id3-writer": "^4.4.0",
     "electron-updater": "^5.2.1",
-    "ffmpeg-static": "4.2.7",
+    "ffmpeg-static": "^5.2.0",
     "fluent-ffmpeg": "^2.1.2",
```

Comment:
I needed to update ffmpeg-static to the latest to be able to do npm i on the "M1" processor, this should be evaluated by build if there are some breaking changes, I checked them but there is none described on the package itself.